### ISSUE TITLE
Doesn't load the .env when the server is starting

### ISSUE THREAD

**Issue by cbaconnier** [codemix/yii2-configloader#1]
Tried with a fresh install of yii2.
When I start the server with `php yii serve`, the .env isn't loaded.
I found this workaround: 

1. renaming the .env file to .envv
2. starting the server with `php yii serve`
3. renaming the .envv to .env

Yii2 (2.0.5)
Windows 10
PHP 7.0.4

Perhaps, I missed something in the configuration?

**Comment by mikehaertl** [comment]
Please check the README first. You have to modify your entry script. The default yii2 app template does not use this extension, so of course this will not work out of the box.

**Comment by cbaconnier** [comment]
Thanks for the answer. I did exactly as described in the point 3 of the readme.
I should mention that the .env does work after the 3 steps I wrote above. 

here's my web/index.php
```
<?php
use codemix\yii2confload\Config;

require(__DIR__ . '/../vendor/autoload.php');
$config = Config::bootstrap(__DIR__ . '/..');
Yii::createObject('yii\web\Application', [$config->web()])->run();
```
and my yii file

```
<?php
use codemix\yii2confload\Config;

require(__DIR__ . '/vendor/autoload.php');
$config = Config::bootstrap(__DIR__);
$application = Yii::createObject('yii\console\Application', [$config->console()]);
exit($application->run());
```

**Comment by mikehaertl** [comment]
Hmm, not sure then. I've never used `yii serve` but it should not really make a difference. As the code of this extension is very simple I'd suggest you add some `print_r(...); exit;` at interesting points to find out what's going on. 

Sorry that I can't provide more help.

**Comment by cbaconnier** [comment]
So, I found the issue.
The $_ENV was returning an empty array while getenv() not. The problem was coming from my php.ini

By default I had: 
`variables_order = "GPCS"`

I changed to:
`variables_order = "EGPCS"`

http://php.net/manual/en/ini.core.php#ini.variables-order `E` will allow $_ENV

Thanks for the help!

**Comment by mikehaertl** [comment]
Hmm, that's interesting. I've only tested this with Apache + PHP module and PHP-FPM. The default seems to be `GPCS`. There's a note in the manual, that `S` implies `E` for fastcgi (PHP-FPM). Now I wonder a bit why it works for Apache + PHP module.

If you have an idea let me know. I further think, that you could report this finding to https://github.com/vlucas/phpdotenv which we use to load the `.env` file. The issue (if there is any) should be fixed there.

**Comment by mikehaertl** [comment]
@cbaconnier To be on the save side I've pushed a fix so that we now use `getenv()` instead of `$_ENV[]` everywhere. Could you maybe test this? The code is on master.

**Comment by cbaconnier** [comment]
@mikehaertl Your last fix seems to work with GPCS now. _(EGPCS as well)_
I didn't test with phpunit, though

About phpdotenv, I have seen some similar issues and basically the answers are to add the 'E' to GPCS or to use getenv()
https://github.com/vlucas/phpdotenv/issues/108
https://github.com/vlucas/phpdotenv/issues/47

**Comment by mikehaertl** [comment]
Ok, thanks. I'll create a 0.9.1 release with this change and will also test if things still work for apache + mod_php.

But I also found this comment [here](http://php.net/manual/en/reserved.variables.environment.php#107413):

> Comments for this page seem to indicate getenv() returns environment variables in all cases.
>
> For getenv() to work, php.ini variables_order must contain 'E'.

This is different from your experience and from other comments. Maybe it only applied to older PHP versions. At least I hope so.

**Comment by mikehaertl** [comment]
Released 0.9.1. Thanks for your help!
